pkg/katautils/utils.go

```diff
@@ -65,7 +65,7 @@ func ResolvePath(path string) (string, error) {
 	if err != nil {
 		if os.IsNotExist(err) {
 			// Make the error clearer than the default
-			return "", fmt.Errorf("file %v does not exist", absolute)
+			return absolute, fmt.Errorf("file %v does not exist", absolute)
```

Comment:
This seems a bit unusual, but since an error *is* being returned is safe. Do we really need this though as the absolute path is already being returned in the error itself?

Reply:
Yes we need this so as to check that path was indeed specified in the configuration.toml file(uncommented). Btw do you mean we should extract absolute path  from the error returned?